using-mobile-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -258,6 +258,260 @@ ACPCore.collectLaunchInfo(userInfo)
 {% endtab %}
 {% endtabs %}
 
+## Logging
+
+The logging APIs allow log messages to be tagged and filtered with the Mobile SDK log messages, and allows the application developer to filter the logged messages based on current logging mode.
+
+As an application developer, use the `setLogLevel` API to filter the log messages coming from the Mobile SDK. When debugging, use  `LoggingMode.VERBOSE` (Android) / `ACPMobileLogLevelVerbose` (iOS) to enable all the logging messages coming from the Mobile SDK and partner extensions. We recommend using a less verbose logging in a production application, for example `LoggingMode.ERROR` (Android) / `ACPMobileLogLevelError` (iOS). 
+
+By default, the Mobile SDK logging mode is set to `LoggingMode.ERROR` (Android) / `ACPMobileLogLevelError` (iOS).
+
+As a Mobile SDK extension developer, use the MobileCore (Android) / ACPCore (iOS) `log` API to include extension log messages with Mobile SDK core log messages.
+
+The Mobile SDK logging modes in order of verbosity, from least to most, are `ERROR`, `WARNING`, `DEBUG`, and `VERBOSE`.
```

Comment:
Please reword so that it reads as follows:

"From least to most verbose, the order of the mobile SDK logging modes is as follows:

* ERROR
* WARNING
* DEBUG
* VERBOSE